com.unity.multiplayer.mlapi/Tests/Runtime/MultiInstance/RPCTests.cs

```diff
@@ -0,0 +1,144 @@
+            var serverClientPlayerResult = new MultiInstanceHelpers.CoroutineResultWrapper<NetworkObject>();
+            yield return MultiInstanceHelpers.Run(MultiInstanceHelpers.GetNetworkObjectByRepresentation((x => x.IsPlayerObject && x.OwnerClientId == clients[0].LocalClientId), server, serverClientPlayerResult));
+
+            // This is the *CLIENT VERSION* of the *CLIENT PLAYER*
+            var clientClientPlayerResult = new MultiInstanceHelpers.CoroutineResultWrapper<NetworkObject>();
+            yield return MultiInstanceHelpers.Run(MultiInstanceHelpers.GetNetworkObjectByRepresentation((x => x.IsPlayerObject && x.OwnerClientId == clients[0].LocalClientId), clients[0], clientClientPlayerResult));
+
+            // Setup state
+            bool hasReceivedServerRPC = false;
+            bool hasReceivedClientRPCRemotely = false;
+            bool hasReceivedClientRPCLocally = false;
+
+            clientClientPlayerResult.Result.GetComponent<RPCTestNetworkBehaviour>().OnClient_RPC += () =>
+            {
+                Debug.Log("ClientRPC received on client object");
+                hasReceivedClientRPCRemotely = true;
+            };
+
+            clientClientPlayerResult.Result.GetComponent<RPCTestNetworkBehaviour>().OnServer_RPC += () =>
+            {
+                // The RPC invoked locally. (Weaver failure?)
+                Assert.Fail("ServerRPC invoked locally. Weaver failure?");
+            };
+
+            serverClientPlayerResult.Result.GetComponent<RPCTestNetworkBehaviour>().OnServer_RPC += () =>
+            {
+                Debug.Log("ServerRPC received on server object");
+                hasReceivedServerRPC = true;
+            };
+
+            serverClientPlayerResult.Result.GetComponent<RPCTestNetworkBehaviour>().OnClient_RPC += () =>
+            {
+                // The RPC invoked locally. (Weaver failure?)
+                Debug.Log("ClientRPC received on server object");
+                hasReceivedClientRPCLocally = true;
+            };
+
+            // Send ServerRPC
+            clientClientPlayerResult.Result.GetComponent<RPCTestNetworkBehaviour>().MyServerRpc();
+
+            // Send ClientRPC
+            serverClientPlayerResult.Result.GetComponent<RPCTestNetworkBehaviour>().MyClientRpc();
+
+            // Wait for RPCs to be received
+            yield return MultiInstanceHelpers.Run(MultiInstanceHelpers.WaitForCondition(() => hasReceivedServerRPC && hasReceivedClientRPCLocally && hasReceivedClientRPCRemotely));
+
+            Assert.True(hasReceivedServerRPC, "ServerRPC was not received");
+            Assert.True(hasReceivedClientRPCLocally, "ClientRPC was not locally received on the server");
+            Assert.True(hasReceivedClientRPCRemotely, "ClientRPC was not remotely received on the client");
+
+            // Release frame rate
+            Application.targetFrameRate = targetFrameRate;
+
+            // Cleanup
+            MultiInstanceHelpers.Destroy();
```

Comment:
It might be useful to move this into a TearDown attribute decorated method so in the event there is a failure the rest of the unit tests that use MultiInstanceHelpers will not error out due to a SIPTransport already started error.